### PR TITLE
Liquebase insert data

### PR DESCRIPTION
**Pull request by jioter**
Insert data into all tables

### CODE REVIEW

File: src/main/resources/db/changelog/init/15062021_create_location_discount_table.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            schemaName: public
+            tableName: location_discount
+            columns:
+              - column:
+                  name: loc_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
+                    primaryKey: true
+                    primaryKeyName: pk_ld_loc_id
```
Comment (dmitriychernov7) [inline src/main/resources/db/changelog/init/15062021_create_location_discount_table.yml]: rename to pk_location_discount_id

File: src/main/resources/db/changelog/init/15062021_create_location_discount_table.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                    foreignKeyName: fk_ld_location
+                    references: location(loc_id)
+              - column:
+                  name: dis_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
+                    primaryKey: true
+                    primaryKeyName: pk_ld_dis_id
```
Comment (dmitriychernov7) [inline src/main/resources/db/changelog/init/15062021_create_location_discount_table.yml]: rename to be the same as pk_location_discount_id

File: src/main/resources/db/changelog/init/15062021_create_user_discount_table.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            schemaName: public
+            tableName: user_discount
+            columns:
+              - column:
+                  name: usr_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
+                    primaryKey: true
+                    primaryKeyName: pk_ud_usr_id
```
Comment (dmitriychernov7) [inline src/main/resources/db/changelog/init/15062021_create_user_discount_table.yml]: rename to pk_user_discount_id

File: src/main/resources/db/changelog/init/15062021_create_user_discount_table.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+                    foreignKeyName: fk_ud_user
+                    references: user(usr_id)
+              - column:
+                  name: dis_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
+                    primaryKey: true
+                    primaryKeyName: pk_ud_dis_id
```
Comment (dmitriychernov7) [inline src/main/resources/db/changelog/init/15062021_create_user_discount_table.yml]: rename to pk_user_discount_id (should be the same as previous)

File: src/main/resources/db/changelog/init/sql/insert_data.sql [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+INSERT INTO category (cat_id, cat_title, cat_image_url, cat_created, cat_modified)
+VALUES (1, 'Sports and fitness', 'http://localhost/images/sport_image.jpeg', current_timestamp, current_timestamp);
+
+INSERT INTO category (cat_id, cat_title, cat_image_url, cat_created, cat_modified)
+VALUES (2, 'Food and entertainment', 'http://localhost/images/food_and_ent_category_image.jpeg', current_timestamp, current_timestamp);
+
+INSERT INTO category (cat_id, cat_title, cat_image_url, cat_created, cat_modified)
+VALUES (3, 'Tourism and leisure', 'http://localhost/images/tourism_and_leisure.jpeg', current_timestamp, current_timestamp);
+
+INSERT INTO "user" (usr_id, usr_first_name, usr_last_name, usr_email, usr_password, usr_created, usr_modified, loc_id, rol_id)
```
Comment (dmitriychernov7) [inline src/main/resources/db/changelog/init/sql/insert_data.sql:46]: add link user to role

File: src/main/resources/db/changelog/init/sql/insert_data.sql [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+VALUES (1, 'Admin', 'Admin', 'admin@gmail.com', 'admin', current_timestamp, current_timestamp, 1, 2);
+
+INSERT INTO "user" (usr_id, usr_first_name, usr_last_name, usr_email, usr_password, usr_created, usr_modified, loc_id, rol_id)
```
Comment (dmitriychernov7) [inline src/main/resources/db/changelog/init/sql/insert_data.sql:49]: assign role to this user

File: src/main/resources/db/changelog/init/sql/insert_data.sql [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+VALUES (2, 'User', 'User', 'test_user@gmail.com', 'test', current_timestamp, current_timestamp, 1, 1);
+
+INSERT INTO "user" (usr_id, usr_first_name, usr_last_name, usr_email, usr_password, usr_created, usr_modified, loc_id, rol_id)
```
Comment (dmitriychernov7) [inline src/main/resources/db/changelog/init/sql/insert_data.sql:52]: assign role to this user

File: src/main/resources/db/changelog/init/15062021_create_location_discount_table.yml (hunk 1 above)
Comment (jioter) [reply src/main/resources/db/changelog/init/15062021_create_location_discount_table.yml]: Changed in other PR

File: src/main/resources/db/changelog/init/15062021_create_location_discount_table.yml (hunk 2 above)
Comment (jioter) [reply src/main/resources/db/changelog/init/15062021_create_location_discount_table.yml]: Changed in other PR

File: src/main/resources/db/changelog/init/15062021_create_user_discount_table.yml (hunk 3 above)
Comment (jioter) [reply src/main/resources/db/changelog/init/15062021_create_user_discount_table.yml]: Changed in other PR

File: src/main/resources/db/changelog/init/15062021_create_user_discount_table.yml (hunk 4 above)
Comment (jioter) [reply src/main/resources/db/changelog/init/15062021_create_user_discount_table.yml]: Changed in other PR

File: src/main/resources/db/changelog/init/sql/insert_data.sql (hunk 5 above)
Comment (jioter) [reply src/main/resources/db/changelog/init/sql/insert_data.sql:46]: There is link role (rol_id)

File: src/main/resources/db/changelog/init/sql/insert_data.sql (hunk 6 above)
Comment (jioter) [reply src/main/resources/db/changelog/init/sql/insert_data.sql:49]: There is link with role (rol_id)

File: src/main/resources/db/changelog/init/sql/insert_data.sql (hunk 7 above)
Comment (jioter) [reply src/main/resources/db/changelog/init/sql/insert_data.sql:52]: There is link role (rol_id)

File: src/main/resources/db/changelog/init/sql/insert_data.sql [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
+VALUES (1, 'Ukraine', 'Kyiv', current_timestamp, current_timestamp);
+
+INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
+VALUES (2, 'Ukraine', 'Lviv', current_timestamp, current_timestamp);
+
+INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
+VALUES (3, 'Ukraine', 'Kharkiv', current_timestamp, current_timestamp);
+
```
Comment (MarvelGa) [inline src/main/resources/db/changelog/init/sql/insert_data.sql:15]: after loc_id values "3" goes  "5". Maybe  loc_id  with values 4 is missed?

Comment (jioter) [reply src/main/resources/db/changelog/init/sql/insert_data.sql:15]: yep, added

**Review by ahlukhouski2 — APPROVED** [review]

**Review by MarvelGa — APPROVED** [review]

**Pull request merged**